src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,20 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
 public class HelloWorld {
+    private static final int SIZE = 6;
 
+    public static void main(String[] args) {
+        FigureSupplier figureSupplier = new FigureSupplier();
+        Figure [] figures = new Figure[SIZE];
+        for (int i = 0; i < SIZE / 2; i++) {
+            figures[i] = figureSupplier.getRandomFigure();
+        }
+        for (int i = SIZE / 2; i < SIZE; i++) {
+            figures[i] = figureSupplier.getdefaultFigure();
+        }
+        for (Figure figure : figures) {
+            figure.draw();
+        }
+
+    }
```

Comment:
Don't use static methods in your solution. Instead of a static main method, you could create an instance of the HelloWorld class and call an instance method to run your code.